packages/relay/src/lib/services/cacheService/cacheService.ts

```diff
@@ -92,7 +92,7 @@ export class CacheService {
 
     this.internalCache = new LocalLRUCache(logger.child({ name: 'localLRUCache' }), register);
     this.sharedCache = this.internalCache;
-    this.isSharedCacheEnabled = process.env.TEST === 'true' ? false : this.isRedisEnabled();
+    this.isSharedCacheEnabled = process.env.TRUE === 'true' ? false : this.isRedisEnabled();
```

Comment:
Is the name of the env. variable correct ?

Reply:
Noope